packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -0,0 +1,365 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
```

Comment:
I don't know what search anchor refers to, I couldn't find a mention in the guidelines

Reply:
Sorry for the confusion. To open a search view, I'm creating a SearchAnchor widget so that we can get all the anchor(SearchBar or IconButton or any widget) information, such as size, position and etc. Here's the design doc https://docs.google.com/document/d/1_WOcgddfJ7OyYyaCJhkIWgOYywj2jDeMooFsKg012B4/edit#

Reply:
Thanks! I'm a bit confused because it talks about the introduction of 2 widgets, and the `SearchView` section only mentions `SearchAnchor`. Are these terms used interchangeably and should s/anchor/view/?

If `SearchView` and `SearchAnchor` are distinct widgets, is there a way to avoid having 3 public widgets? 2 would be ideal.

Reply:
I see. I think we will only need `SearchBar` and `SearchAnchor` widgets. Will update the design soon! Thanks:)